src/index.js

```diff
@@ -53,7 +64,9 @@ const reduxStore = initializeReduxStore(routerHistory);
 ReactDOM.render(
   <Provider store={reduxStore}>
     <ConnectedRouter history={routerHistory}>
-      <AuthRoute path={RoutePaths.ROOT} component={AppContainer} />
+      <ScrollToTop>
```

Comment:
Conceptually, this doesn't belong here. A better place would be in `FormContainer`.